### ISSUE TITLE
Unclear how to contribute to docs

### ISSUE THREAD

**Issue by theadammurphy** [AIStream-Peelout/flow-forecast#355]
This is a great library with a lot of potential. I would love to help out and fix the myriad of things that need fixing. But I have no idea how!

None of the .rst files in docs/ contain anything similar to the Confluence of ReadTheDocs docs. 

The [Contributing](https://flow-forecast.atlassian.net/wiki/spaces/FF/pages/11403276/Contributing) page also says nothing about how to contribute to Confluence or ReadTheDocs. 

Can you show us how to contribute to:
a) Confluence, and 
b) ReadTheDocs

**Comment by isaacmg** [comment]
Yes in the future there should be a Confluence on contributing to docs. But in short:
(a) Confluence you need to [sign up](https://id.atlassian.com/signup?continue=https%3A%2F%2Fflow-forecast.atlassian.net%2Flogin%3FredirectCount%3D1%26dest-url%3D%252Fwiki%252Fhome%26application%3Dconfluence&application=confluence). Once you are signed up permission can be granted by myself or another project admin to edit. The actual editing follows a basic WYSIWYG editor.
(b) ReadTheDocs is built using the literal docstrings in the code along with the .rst files in the documents directory. In order for docs to be built they will need to be merged via  PR (like any other code).

**Comment by theadammurphy** [comment]
I've just made some updates to the [Training Models](https://flow-forecast.atlassian.net/wiki/spaces/FF/pages/364019713/Training+Models) page. 

I thought there would be a review process but it looks like they have been published already. Is this an accident?

Main changes:
1. Made all code blocks 'Python' code blocks - now we have pretty code highlighting!
2. Made all code blocks follow Python syntax (all indents are correct and consistent dict definition style)
3. Added brief explanation about model names and where to find the list of all models currently supported.
